README.md

```diff
@@ -72,6 +72,26 @@ To clear the local history of updates:
 
  Only updates that were distributed from the LVFS will be reported to the LVFS.
 
+Enterprise Use
+--------------
+
+The flow of updates can be controlled in the enterprise using the
+"approved updates" feature. This allows the domain administrator to filter
+the possible updates from a central server (e.g. the LVFS) to only firmware
+that have been tested specifically in your organisation.
+
+The list of approved updates can be enabled by adding `ApprovalRequired=true`
+to the remote configuration file, e.g. `lvfs.conf`. Once enabled, the list
+of approved updates can be set in `daemon.conf`, for instance:
+
+    ApprovedFirmware=foo,bar
+
+The `foo,bar` refers to the container checksum as specified in the metadata
+file.
```

Comment:
Rewording proposal.
One enabled, the list of approved updates can be set in `daemon.conf` using a comma delimited list.
For example:

```
ApprovedFirmware=foo,bar
```

`foo,bar` refers to the container checksums that would correspond to two updates in the metadata file.